model.js

```diff
@@ -2,8 +2,43 @@
  * @fileoverview the model class that stores data and handles all data interactions for the app.
  */
 
+import * as d3 from "d3";
+import { nest } from 'd3-collection';
 import Papa from "papaparse";
 
+const bizsector_codes = {
+  1: "Accommodation and catering",
+  2: "Agriculture, forestry, animal husbandry, fisheries",
+  3: "Construction industry",
+  4: "Culture, sports and entertainment",
+  5: "Education",
+  6: "Financial industry",
+  7: "Health and social work",
+  8: "Information transmission, software and information technology services",
+  9: "Leasing and business services",
+  10: "Manufacturing industry",
+  11: "Mining, oil and gas industry",
+  12: "Production and supply of electricity, heat, gas and water",
+  13: "Real estate industry",
+  14: "Residential services, repair and other services",
+  15: "Scientific research and technological services",
+  16: "Tourism",
+  17: "Transport, storage and postal industry",
+  18: "Water, environment and public facilities management",
+  19: "Wholesale and retail trade",
+  0: "Other",
+};
+
+const revchange_codes = {1: "+ >30%",
```

Comment:
Nit: Could we keep these constants in a dedicated constants.js file and then export each one like "export const REVCHANGE_CODES = ..."?